### ISSUE TITLE
Initial configuration creation fails

### ISSUE THREAD

**Issue by gabixdev** [KyokoBot/kyoko#68]
```
[16:10:51.850] [ERROR] [main] Error creating default configuration file!
com.fasterxml.jackson.databind.JsonMappingException: (was java.lang.IllegalArgumentException) (through reference chain: moe.kyokobot.shared.Settings["bot"]->moe.kyokobot.shared.Settings$BotSettings["error-color"])
        at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:394)
...
        at com.fasterxml.jackson.databind.ObjectMapper.writeValueAsString(ObjectMapper.java:3219)
        at moe.kyokobot.shared.util.JsonUtil.toJSON(JsonUtil.java:24)
        at moe.kyokobot.bot.Main.loadSettings(Main.java:135)
        at moe.kyokobot.bot.Main.main(Main.java:56)
Caused by: java.lang.IllegalArgumentException: null
        at java.base/java.lang.String.repeat(Unknown Source)
        at moe.kyokobot.shared.Settings.encodeColor(Settings.java:181)
        at moe.kyokobot.shared.Settings$BotSettings.internal$errorColor(Settings.java:152)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```

**Comment by gabixdev** [comment]
Should get fixed by ec6b4fbecc431e79091c6ad0b122e9159bc71dc4

**Comment by gabixdev** [comment]
Fixed
